src/chrome/content/rules/DNC.org.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	See also Democrats.org.xml.
+
```

Comment:
Both https://gems.dnc.org and the HTTP version time out for me, so I don't think we should include or even mention it.